### ISSUE TITLE
mdstcl on OS X

### ISSUE THREAD

**Issue by amicitas** [MDSplus/mdsplus#1485]
I am unable to run `mdstcl` on my current OS X installation (10.14.1 Mojave).  

When running the following command on the command line:
```
/usr/local/mdsplus/bin/mdstcl
```

I get the following error:

```
I/O warning : failed to load external entity "./mdsdcl_commands.xml"
 Error: unable to parse ./mdsdcl_commands.xml
```

**Versions Tested:**

-7.59.1 result: Failure
-7.49.0 result: Failure

**System:**
    OS X 10.14.1 (Mojave)

Installation using the standard OS X package.

**Comment by joshStillerman** [comment]
You probably do not have MDSPLUS_DIR environment variable defined.
did you follow the steps:
http://www.mdsplus.org/index.php/Latest_Macintosh_Distributions#Post_Installation_Configuration
if you type:
```
echo $MDSPLUS_DIR
```

**Comment by amicitas** [comment]
You were correct about the issue being the MDSPLUS_DIR environmental variable.  I did indeed follow the instructions, which say that my start up scripts would be automatically modified by the installer (I did not think to check if the modification was successful.)

The problem was that my system was not running the `/usr/local/mdsplus/setup.sh` script.  Once this script is loaded though standard bash resolution sequence (or is manually sourced) 'mdstcl' works as expected.
